### PR TITLE
updating README to recommend v2 installers

### PR DESCRIPTION
**Pull request by nateprewitt**
Updating README to recommend pre-built installers for AWS CLI v2.

### CODE REVIEW

**Review by timoschilling — CHANGES_REQUESTED** [review]
all links aren't working (cloudfront 403)

**Comment by nateprewitt** [comment]
Thanks @timoschilling! We're currently getting the routing setup, we'll merge the PR after that's completed.

File: README.rst [hunk 1]
```diff
@@ -38,37 +38,13 @@ this `blog post <https://aws.amazon.com/blogs/developer/deprecation-of-python-2-
 Installation
 ------------
 
-The easiest way to install aws-cli is to use `pip`_ in a ``virtualenv``::
+AWS CLI v2 can easily be installed on most standard platforms:
 
-    $ pip install awscli
+* `MacOS executable installer <https://awscli.amazonaws.com/AWSCLIV2.pkg>`__
```
Comment (stealthycoin) [inline README.rst]: MacOS pkg installer

Comment (nateprewitt) [reply README.rst]: Good call, updated.

**Review by kyleknap — APPROVED** [review]
Nice! Looks good. :ship:

**Review by stealthycoin — APPROVED** [review]

**Pull request merged**